docs/topics/auth/passwords.txt

```diff
@@ -693,6 +693,40 @@ or if you have API calls that allow passwords to be set, for example.
 Note that where the password is passed to one of these functions, this should
 always be the clear text password - not a hashed password.
 
+If you would like to override the default error messages if a password fails
+to validate, you can subclass the built-in validator classes and override the
+``get_error_message()`` method.
+
+.. code-block:: python
+
+    class CustomMinimumLengthValidator(MinimumLengthValidator):
+
+        def get_error_message(self):
+            return "Custom error message for minimum length validation"
+
+.. code-block:: python
+
+    class CustomUserAttributeSimilarityValidator(UserAttributeSimilarityValidator):
+
+        def get_error_message(self, verbose_name):
+            return "Custom error message for attribute similarity validation"
```

Comment:
The example may be even more helpful if we include ``verbose_name`` in the return line

Reply:
Thanks, I included it in the return line.